src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -488,3 +488,49 @@ def test_bz1927195(multihost, backupsssdconf, proxy_sleep):
                                         "queuing request' "
                                         "/var/log/sssd/sssd_example1.log")
         assert result.stdout_text == '1\n'
+
+    @staticmethod
+    def test_netgroup(multihost, backupsssdconf):
+        """
+        :title: Lookup alias and original netgroups
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=804103
```

Comment:
Hi,

it this the right bugzilla link? It is currently pointing to a nss-pam-ldapd ticket about missing netgroups?

bye,
Sumit

Reply:
I will remove that link , just mentioned  as it was one the bzs mentioned in bash script